src/edu/csus/ecs/pc2/core/execute/Executable.java

```diff
@@ -471,6 +481,11 @@ public IFileViewer execute(boolean clearDirFirst) {
                 int dataSetNumber = 0;
                 boolean passed = true;
 
+                // create somewhat secure ndjson file name for per test case execution results.
+                // format of the name is: RSSexecuteinfo.ndjson where R = run number, SS = current clock seconds
+                // eg: for run 453 at 12:00:09:   4539executeinfo.ndjson
```

Comment:
I think this format will be confusing to anyone who looks at the folder containing the files and tries to figure out the logic behind the file naming;   `4539` looks (something like) a `RunId`... but I would NEVER guess that I should split off the last digit and treat `453` and `9` as separate fields.  Wouldn't it be more helpful to do something like insert periods between the fields -- e.g. `453.9.executeinfo.ndjson`?  Or even better, `run493.time9.executeinfo.ndjson`?

Reply:
I modeled the format after the security results file that is currently created by PC2 (`#sssXRSAM.#`). (see `Executable.validateProgram()`.  It is mean to be obscure to avoid someone guessing it.   This file is not used yet.  It will hold the test case summary results.  It is current a replacement variable (`{:executeinfofilename}`) that we can easily add later to the validator command line.

Reply:
> I modeled the format after the security results file that is currently created by PC2 (`#sssXRSAM.#`). (see `Executable.validateProgram()`. It is mean to be obscure to avoid someone guessing it. ...

Ok, good point.  Although I worry that it may be even "too obscure for US"...  Feel free to mark this comment "Resolved" if you wish.

Reply:
No code changes.